service/surf/apps/materials/tests/tests_material_meta_tags.py

```diff
@@ -0,0 +1,61 @@
+from django.test import SimpleTestCase, Client
+from unittest.mock import patch
+
+import surf.apps.materials.views as views
+
+
+def patched_version_with_material(a, b):
+    return [{
+        'title': 'foo',
+        'description': 'bar'
+    }]
+
+
+def patched_version_without_material(a, b):
+    return []
+
+
+@patch('webpack_loader.loader.WebpackLoader.get_bundle')
+class TestMaterialMetaTags(SimpleTestCase):
+    def setUp(self):
+        self.original = views._get_material_by_external_id
```

Comment:
This kind of swapping is what @patch can do for you.

Reply:
Yeah, I know. However this is a private method on a module level. You can't patch that, you can only monkey patch it... (as far as I could find).

Reply:
Ok, never heard of that limitation. It can be public though.